package.json

```diff
@@ -64,6 +64,7 @@
     "dompurify": "^2.3.10",
     "electron-updater": "^5.2.1",
     "express": "^4.18.1",
+    "@inigolabs/ffi-napi": "^4.0.10",
```

Comment:
Given the… opaqueness of this package, we should probably not use it. We may need to reconsider alternative ways to get the certificates (such as have a native executable that generates a tar file directly, instead of doing it from JS). :(